crates/core/src/remappings.rs

```diff
@@ -197,7 +223,7 @@ fn generate_remappings(
     if soldeer_config.remappings_regenerate {
         let dependencies = read_config_deps(&paths.config)?;
         new_remappings = remappings_from_deps(&dependencies, paths, soldeer_config)?
-            .into_iter()
+            .into_par_iter()
```

Comment:
Not sure it's beneficial here, there are only few remappings (roughly the same number as top-level dependencies). Same remark for the sort below. Would love some benchmarks.